### PR TITLE
Add monitor for music play

### PR DESCRIPTION
**Pull request by gavin-lin**
1. Done by monitoring setMedia() in write events.
2. The function name 'write' is really misleading, but since we are going to make output more readable, we can deal with this that time.
3. Because of item 2, I don't want too many write events show up in other apps, but I have not found good way to sort them yet, so I just filter them out except the one with setMedia().

### CODE REVIEW

**Comment by gavin-lin** [comment]
This will fix issue #92

File: utils/watcher_strace.py [hunk 1]
```diff
@@ -107,6 +113,20 @@
                                         item)
                                     break
                 # For file access, put it here to ignore sendmsg
+                # For music file being set to play
+                elif 'setMedia()' in output:
```
Comment (Cypresslin) [inline utils/watcher_strace.py:117]: I think you can move this part to group with line 68

First check if it's a write event,
Check if it's the desired event setMedia()
    Yes -> call printer
    No -> continue the loop

**Comment by gavin-lin** [comment]
I was thinking to separate supressor from other codes, so if we need more write events, we can modify the supressor easily.

File: utils/watcher_strace.py (hunk 1 above)
Comment (gavin-lin) [inline utils/watcher_strace.py:117]: I was thinking to separate supressor from other codes, so if we need more write events, we can modify the supressor easily.

**Comment by Cypresslin** [comment]
Make sense, thank you.

**Pull request merged**